mobius/src/unit_complex.rs

```diff
@@ -0,0 +1,46 @@
+use std::ops::Neg;
+
+use crate::{complex_error::ComplexError, Complex};
+
+#[derive(PartialEq, Clone, Copy, Debug, derive_more::Display)]
+pub struct UnitComplex(Complex);
+
+impl UnitComplex {
```

Comment:
Add some unit tests for this struct:

- `normalize()` throws error for zero vector or infinity
- `normalize()` computes a unit length vector
- `from_angle()` computes a vector from cos/sin
- `rot90()` rotates the vector ccw
- `rot90()` repeated 4 times gives the same vector
- `-z` returns the negated vector
- `rot90().rot90()` is the same as `-z`